slacktest/server.go

```diff
@@ -61,8 +63,10 @@ func NewTestServer(custom ...Binder) *Server {
 	s.Handle("/bots.info", botsInfoHandler)
 	s.Handle("/auth.test", authTestHandler)
 	s.Handle("/reactions.add", reactionAddHandler)
+	s.Handle("/apps.connections.open", appsConnectionsOpenHandler)
 
 	httpserver := httptest.NewUnstartedServer(s.mux)
+	httpserver.Config.Handler = logger.New().Handler(httpserver.Config.Handler)
```

Comment:
Is this necessary? I'd like to get rid of the `logger` package. It may be easy enough to implement a few lines to replace it. But between adding the package to have some logs, or keeping it out and not have one more package, I'd prefer the latter.